mailet/base/src/test/java/org/apache/mailet/base/test/MimeMessageBuilder.java

```diff
@@ -192,6 +192,10 @@ public static MimeMessage mimeMessageFromStream(InputStream inputStream) throws
         return new MimeMessage(Session.getDefaultInstance(new Properties()), inputStream);
     }
 
+    public static MimeMessage mimeMessageFromBytes(byte[] bytes) throws MessagingException {
+        return new MimeMessage(Session.getDefaultInstance(new Properties()), new ByteArrayInputStream(bytes));
```

Comment:
why don't you call mimeMessageFromStream instead ?

Reply:
too fast copy/paste, done (and sorry for the push -f)